docs/static/releasenotes.asciidoc

```diff
@@ -36,6 +37,67 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-8-0]]
+=== Logstash 8.8.0 Release Notes
+
+[[notable-8.8.0]]
+==== Notable issues fixed
+
+* In previous Logstash versions, when a pipeline used Elasticsearch output and the credentials expired, it ended in the
+  pipeline remained stuck in disconnecting from an invalidated Elasticsearch endpoint also when the pipeline definition
+  was updated with the new connection credentials.
+  The issue https://github.com/elastic/logstash/issues/14739[14739] describe how to reach such condition in terms of Central Pipeline Management.
+  To fix this problem, Logstash core has been modified so that a plugin can request a termination of the inflight batch
+  during the shutdown of itself.
+  The bugfix https://github.com/elastic/logstash/pull/14940[#14940] solves the original issue,
+  avoiding any data loss when used in combination with persistent queues.
+
```

Comment:
Maybe something like:

Fix a race condition that prevents Logstash from updating a pipeline's configuration with in-flight events experiencing connection errors. https://github.com/elastic/logstash/issues/14739[14739]

This issue primarily manifests following the update of Elasticsearch credentials through Central Management, after they expired while events were in-flight. It causes the Elasticsearch Output to get stuck attempting to send events with the expired credentials instead of using the updated ones.

To address this problem, Logstash has improved the pipeline shutdown phase, to allow the output plugin to request the termination of the in-flight batch of events, preventing the need for administrators to manually restart Logstash. Furthermore, when used in combination with a persistent queue to avoid data loss, the batch is eligible for reprocessing on pipeline restart.

Plugin developers can also decide to make use of such functionality on other output plugins.

Reply:
Fuxed with d8225a431f99deb0545fceb70aa13051a69505ba